ufc/bandit-flags-v1.json

```diff
@@ -1,5 +1,6 @@
 {
     "createdAt": "2024-04-17T19:40:53.716Z",
+    "format": "SERVER",
```

Comment:
I find `server`/`client` dichotomy misleading here.

1. It's not obvious what it means. I had to look up the code to confirm it's about obfuscation.
2. Clients can work with non-obfuscated configs, and servers may work with obfuscated ones

Is there any reason we prefer "format" over straightforward `"obfuscated": true`?

Reply:
I still don't understand the reason to be vague or how this helps